### PR TITLE
[CDAP-14361] Modifies header product dropdown to not show cdap mode for now

### PR DESCRIPTION
**Pull request by ajainarayanan**
- Fixes header to show cog-wheel icon for product dropdown instead of cdap mode
- Fixes `redirect-to-login` service to rightly account for logout behavior

JIRA: https://issues.cask.co/browse/CDAP-14361
Build: https://builds.cask.co/browse/CDAP-URUT104

### CODE REVIEW

File: cdap-ui/app/cdap/components/Header/ProductDropdown/index.js [hunk 1]
```diff
@@ -143,13 +141,7 @@ export default class ProductDropdown extends Component {
...
           <DropdownToggle caret>
             <div className="secure-mode-icon">
-              <IconSVG name={ window.CDAP_CONFIG.securityEnabled ? "icon-lock_close" : "" } />
-              <div className="cdap-mode">{mode}</div>
-            </div>
-            <div className="cdap-logo-container">
-              <div className="caret-down-container">
-                <IconSVG name="icon-caret-down" />
-              </div>
+              <IconSVG name="icon-cog" />
```
Comment (bdmogal) [inline cdap-ui/app/cdap/components/Header/ProductDropdown/index.js]: we're using the multiple cog wheel right?

Comment (ajainarayanan) [reply cdap-ui/app/cdap/components/Header/ProductDropdown/index.js]: I see. ok changing it

File: cdap-ui/app/cdap/services/redirect-to-login.js [hunk 2]
```diff
@@ -17,9 +17,12 @@
 import cookie from 'react-cookie';
 export default function RedirectToLogin(data) {
   let {statusCode} = data;
-  let {url} = data.resource;
+  let {url} = data.resource || {};
```
Comment (elfenheart) [inline cdap-ui/app/cdap/services/redirect-to-login.js:20]: shouldn't this be `data.resource || ''`?

Comment (ajainarayanan) [reply cdap-ui/app/cdap/services/redirect-to-login.js:20]: No. `resource` is an object with url, header and other properties that we pass to express server.

**Review by elfenheart — APPROVED** [review]
👍 LGTM

**Comment by ajainarayanan** [comment]
Latest green build: https://builds.cask.co/browse/CDAP-URUT104-3

**Pull request merged**